app/controllers/sections/journeyType/HowMovementTransportedController.scala

```diff
@@ -101,7 +101,7 @@ class HowMovementTransportedController @Inject()(
       userAnswers.remove(JourneyTypeSection).resetIndexedSection(TransportUnitsSection, Index(0)).set(
         TransportUnitTypePage(Index(0)), FixedTransport
       )
-    case userAnswers if userAnswers.get(HowMovementTransportedPage).contains(FixedTransportInstallations) =>
+    case userAnswers if userAnswers.getFromUserAnswersOnly(HowMovementTransportedPage).contains(FixedTransportInstallations) =>
```

Comment:
Why?

Reply:
This is because it should only clear down if the value in saved answers has changed; as the answer in the IE801 could be `FixedTransportInstallations` which means it would clear down stuff unnecessarily.